corgidrp/caldb.py

```diff
@@ -26,7 +26,11 @@
     "EXCAMT",
 ]
 
-labels = {data.Dark: "Dark", data.NonLinearityCalibration: "NonLinearityCalibration", data.BadPixelMap: "BadPixelMap", "KGain": data.KGain, }
+labels = {data.Dark: "Dark", 
+          data.NonLinearityCalibration: "NonLinearityCalibration", 
+          data.BadPixelMap: "BadPixelMap", 
+          data.KGain : "KGain", 
+          data.DetectorParams : "DetectorParams"}
```

Comment:
Not a requirement but just a thought: Is there a way to combine this dictionary with the `datatypes` dictionary from data.py. They look like they do very similar things, but are maybe the inverse of each other. If possible having one would be great.